INSTALL.md

```diff
@@ -91,35 +85,29 @@ or follow these instructions manually:
        ## If java is installed:
        make -C dafny runtime
```

Comment:
np: Unrelated to this change, but re-reading over the instructions above, we're missing a `cd` (see Linux instructions above for correct steps that also need to be done for Mac).

Specifically, `cd dafny` before `brew install pre-commit`. Also, we would need to update `make -C dafny runtime` to be`make runtime`, I believe.

Currently, `pre-commit install` is being run in `BASE-DIRECTORY`, which would fail because it's not within the actual `dafny` checked out repo.

Aka, we need:
```
dotnet build dafny/Source/Dafny.sln
cd dafny
brew install pre-commit
pre-commit install
## If java is installed:
make runtime
```
(I believe)

Reply:
Good eyes! I updated the instructions.